src/python/WMCore/WorkQueue/Policy/Start/StartPolicyInterface.py

```diff
@@ -274,6 +278,36 @@ def getDatasetLocations(self, datasets, account=None):
                                                              account=account)
             result[datasetPath] = self.cric.PNNstoPSNs(locations)
         return result
+    
+    def getDatasetLocationsFromMSPileup(self, datasetsWithDbsURL):
+        """
+        Returns a dictionary with the location of the datasets according to MSPileup
+        The definition of "location" here is a union of all sites holding at least
+        part of the dataset (defined by the DATASET grouping).
+        :param datasets: list of datasets
+        :param account: rucio account to use, if it is not provided we fallback to
+        pileup account for backward compatibility
+        :return: a dictionary of dataset locations, key'ed by the dataset name
+        """
+        if isinstance(datasetsWithDbsURL, str):
+            datasetsWithDbsURL = [datasetsWithDbsURL]
+        result = {}
+        for dataset, dbsUrl in zip(datasetsWithDbsURL):
```

Comment:
And this line gives me a ValueError exception.

Reply:
I'm going to commit a new version, were L292 and L293 are deleted, as well as the `zip` in L295